src/javase02/task02/Chancellery.java

```diff
@@ -0,0 +1,44 @@
+package javase02.task02;
+
+public class Chancellery {
```

Comment:
if you assume that Chancellery is abstract class, append 'abstract' specifier

Reply:
I wont do this cause it wil forbid making instances of it